### PR TITLE
Deployed on Heroku

### PR DESCRIPTION
**Pull request by bhavyaagg**
Solves #24 
Running At : https://whispering-garden-72461.herokuapp.com/signup

### CODE REVIEW

**Comment by championswimmer** [comment]
wouldn't you require to specify the Heroku Database name and port etc ? 
In the db/models.js

**Comment by bhavyaagg** [comment]
Bhaiya uss vaali mai kardia and secrets.json use kardi vaha. Secrets.json toh yaha nahi daalni. Deploy karte vakt secret-sample.json ki jagah secret.json use karli. Models mai secret-sample.json se hee use kia hua hai.@championswimmer

**Comment by bhavyaagg** [comment]
Deploy karte vakt secret-sample.json ki jagah secret.json use karli

**Comment by bhavyaagg** [comment]
@championswimmer Updated the PR with the changes in models.js

File: server.js [hunk 1]
```diff
@@ -41,6 +41,6 @@ app.use('/api', apirouter);
 
 app.use('/', express.static(__dirname + '/public_html'));
 
-app.listen(4000, function () {
+app.listen(process.env.PORT||4000, function () {
     console.log("Listening on 4000");
```
Comment (championswimmer) [inline server.js]: fix same in 2nd line too

File: db/models.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 const Sequelize = require('sequelize');
 const secret = require('./../secret-sample.json');
```
Comment (championswimmer) [inline db/models.js]: require from secrets, not secrets-sample

File: db/models.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
-const db = new Sequelize(secret.DATABASE, secret.DB_USER, secret.DB_PASSWORD, {
```
Comment (championswimmer) [inline db/models.js]: require from secrets, not secrets-sample

**Review by championswimmer — CHANGES_REQUESTED** [review]

**Comment by bhavyaagg** [comment]
@championswimmer Updated the PR

File: db/models.js (hunk 2 above)
Comment (bhavyaagg) [reply db/models.js]: Done

**Comment by bhavyaagg** [comment]
Done Finally i hope @championswimmer

**Comment by bhavyaagg** [comment]
Done all the changes

**Comment by bhavyaagg** [comment]
Please review @championswimmer

**Comment by bhavyaagg** [comment]
Please review @championswimmer

**Review by championswimmer — APPROVED** [review]

**Pull request merged**